package/yast2-bootloader.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Wed Jul  5 08:56:43 CEST 2017 - schubi@suse.de
+
+- AuotYaST configuration: Set StorageDevices flag disks_valid
+  to true. So libstorage can scan valid disks and "/" partition
+  can be found. (bnc#1046738)
```

Comment:
Since we have to merge this to SP3 and master branches, to keep the branches diff in good shape, I would also add a reference to bcn#1043132 (the original bug, this fix is just a backport) here.